payment-methods.adoc

```diff
@@ -0,0 +1,285 @@
+2. the payment account has chargeback risk (SEPA, SEPA Instant, Interac e-Transfer, Zelle, Revolut, Chase QuickPay, Popmoney, MoneyBeam, Uphold, and any kind of bank transfer)
+
+In major national currency markets, account signing is not required to enable account aging for face-to-face trading, cash deposits, money orders (US Postal Money Orders, MoneyGram, Western Union) or Advanced Cash.
+
+In other markets, account signing is not currently required for any payment methods.
+
+=== How To Have a Payment Account Signed
+
+A signed payment account indicates that it can be trusted to engage in trades honestly, and as a result, is allowed higher trading limits.
+
+WARNING: Account signing does not _guarantee_ problem-free trading. Instead, it creates a trusted network of accounts where it is _unlikely_ to encounter a scammer.
+
+**Please note that account signing works per payment account.** Having one payment account signed does not affect trading limits on your other payment accounts. You need to go through the account signing process below for each payment account you want to lift limits for.
+
+Here's how to have a payment account signed for higher bitcoin buying limits.
+
+1. **Take an offer to buy bitcoin.** This offer must be from a trading peer with a signed payment account who is able to sign other payment accounts.
++
+If a trading peer is capable of signing your payment account, they'll have a check mark on their offer listing in the `Time since signing` column.
++
+image::peer-can-sign-offer-listing.png[Look for this indicator.,400,400]
++
+You can click the trading peer's avatar to see more details on the account signing.
++
+image::account-signing-details-pop.png[Look for this indicator.,400,400]
++
+[NOTE]
+.How were the first payment accounts signed if there were no peers to sign them?
+====
+Bisq arbitrators signed a collection of accounts that met certain criteria to bootstrap the network of trusted payment accounts just before the v1.2 release.
+
+The following criteria were determined to balance high integrity with volume (so that there were a sufficient number of signed peer accounts available to sign other peer accounts in the beginning):
+
+* payment account must be older than 60 days
+* payment account must have been involved in a dispute as a bitcoin buyer, and received bitcoin as an outcome of that dispute (thereby proving that a successful fiat _payment_ took place)
+
+All payment accounts that met the 2 criteria above were signed by arbitrators and immediately able to sign other payment accounts upon the release of v1.2.
+====
+
+2. **Successfully settle the trade.** When you send your fiat payment to the seller, your payment account will be signed when the seller clicks the `Confirm payment received` button.
++
+Thirty days after signing, the following will take place:
++
+--
+* the 0.01 BTC trading limit will be lifted
```

Comment:
Perhaps mention that it's the buy limit, but indeed both buy and sell limit will increase.